packages/dso-toolkit/src/components/modal/modal.mixins.scss

```diff
@@ -140,12 +112,18 @@
     }
   }
 
+  @media screen and (max-width: media-query-breakpoints.$screen-xxs-max) {
+    .dso-dialog {
+      height: 100vh;
+      max-height: 100vh;
```

Comment:
`dvb` (dynamic viewport block) is een beter alternatief als unit, omdat die rekening houdt met in- en uitschuivende browserpanels. https://caniuse.com/mdn-css_types_length_viewport_percentage_units_dynamic